CameraInspector/CameraInspector.vcxproj

```diff
@@ -0,0 +1,151 @@
+  </ImportGroup>
+  <PropertyGroup Label="UserMacros" />
+  <PropertyGroup />
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <ClCompile>
+      <WarningLevel>Level3</WarningLevel>
+      <Optimization>Disabled</Optimization>
+      <SDLCheck>true</SDLCheck>
+      <AdditionalIncludeDirectories>D:\~SS\~SB\01\SecureBrowser\CameraInspector;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <ClCompile>
+      <WarningLevel>Level3</WarningLevel>
+      <Optimization>Disabled</Optimization>
+      <SDLCheck>true</SDLCheck>
+      <AdditionalIncludeDirectories>D:\~SS\~SB\01\SecureBrowser\CameraInspector;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
+    <ClCompile>
+      <WarningLevel>Level3</WarningLevel>
+      <Optimization>MaxSpeed</Optimization>
+      <FunctionLevelLinking>true</FunctionLevelLinking>
+      <IntrinsicFunctions>true</IntrinsicFunctions>
+      <SDLCheck>true</SDLCheck>
+      <AdditionalIncludeDirectories>D:\~SS\~SB\01\SecureBrowser\CameraInspector;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
+    <Link>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <OptimizeReferences>true</OptimizeReferences>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <ClCompile>
+      <WarningLevel>Level3</WarningLevel>
+      <Optimization>MaxSpeed</Optimization>
+      <FunctionLevelLinking>true</FunctionLevelLinking>
+      <IntrinsicFunctions>true</IntrinsicFunctions>
+      <SDLCheck>true</SDLCheck>
+      <AdditionalIncludeDirectories>D:\~SS\~SB\01\SecureBrowser\CameraInspector;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
+    <Link>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <OptimizeReferences>true</OptimizeReferences>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="FaceDetector.cpp" />
+    <ClCompile Include="FileSystemFrameSaver.cpp" />
+    <ClCompile Include="Frame.cpp" />
+    <ClCompile Include="PhotoMaker.cpp" />
+    <ClCompile Include="Source.cpp" />
+    <ClCompile Include="WebCameraCapture.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="FaceDetector.h" />
+    <ClInclude Include="FileSystemFrameSaver.h" />
+    <ClInclude Include="Frame.h" />
+    <ClInclude Include="IFrameHandler.h" />
+    <ClInclude Include="IFrameSaver.h" />
+    <ClInclude Include="PhotoMaker.h" />
+    <ClInclude Include="WebCameraCapture.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <Image Include="screenshot.jpg" />
```

Comment:
IDK, why it was here)
Fixed: https://github.com/LvCPP/SecureBrowser/commit/bb647818619248e383c20653f6f54b8551bab9a2